docroot/themes/humsci/humsci_basic/src/scss/utilities/_wysiwyg-text-area.scss

```diff
@@ -338,6 +339,16 @@ a[href*="//"].hs-external-link,
       }
     }
   }
+  
+  .ck-style-grid__button & {
+    &::after {
+      width: 18px;
+      height: 11px;
+      margin-bottom: 0;
+      background-size: 11px;
+      padding-left: 0;
+    }
+  }
```

Comment:
This file is used for the styles for the WYSIWYG text blocks in the FE theme. The styles you're adding are only for the CKEditor UI, so please move them in the admin theme CSS (`docroot/themes/humsci/su_humsci_gin_admin/dist/su_humsci_gin_admin.css`).

Also, please add a comment to explain why are you resizing the icon.